agcaballero/portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,94 @@
+/* General */
+
+body {
+  font-family: 'Helvetica Nueue', Helvetica, Arial, sans-serif;
+  background-color: #f4f1de;
+
+  /*
+  width: 70%;
+  margin: 0 auto;
+  */
```

Comment:
Did you want to keep this comment, or can it be removed now?

Reply:
I am removing it.